index.html

```diff
@@ -6582,92 +6595,136 @@ <h2 id="instantiation-of-AudioWorkletNode-and-AudioWorkletProcessor">
             <a>AudioWorkletNode</a> instance is destroyed.
           </p>
           <p>
-            When <a>AudioWorkletNode</a>(<var>context</var>, <var>name</var>,
-            <var>options</var>) constructor is called, the user agent must run
-            the following steps <a data-lt="atomic">atomically</a>.
+            Note that the instantiation of these two objects spans the control
+            thread and the rendering thread.
+          </p>
+          <p>
+            When <a>AudioWorkletNode</a>(<var>context</var>,
+            <var>nodeName</var>, <var>options</var>) constructor is invoked,
+            the user agent must perform the following steps on the control
+            thread, where the constructor was called.
           </p>
           <ol>
             <li>Let <var>this</var> be the instance being created by
             constructor of <a>AudioWorkletNode</a> or its subclass.
             </li>
-            <li>If <var>name</var> does not exists as a key in the
-            <a>AudioWorkletGlobalScope</a>’s <a>node name to processor
-            definition map</a>, throw a <code>NotSupportedError</code>
-            exception and abort these steps.
+            <li>If <var>nodeName</var> does not exists as a key in the
+            <a>BaseAudioContext</a>’s <a>node name to parameter descriptor
+            map</a>, throw a <code>NotSupportedError</code> exception and abort
```

Comment:
Seems like we digressed. Is there anything to be fixed in this PR? I think the current spec text here has a clear distinction between the main thread and the worklet thread. So adding a `synchronous` icon seems redundant to me.